google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -63,8 +66,8 @@ public class SessionPoolOptions {
   @Deprecated private final long initialWaitForSessionTimeoutMillis;
 
   private final boolean autoDetectDialect;
-  private final Duration waitForMinSessions;
-  private final Duration acquireSessionTimeout;
+  private final java.time.Duration waitForMinSessions;
```

Comment:
nit: We can import `java.time.Duration` so that in future we don't have a confusion on which class to use

Reply:
and also it avoids so many changes in the code which is not really necessary.

Reply:
Done